docs/source/configuration.rst

```diff
@@ -780,6 +780,16 @@ keepalive
 
 Determines whether Converse will attempt to keep you logged in across page loads.
 
+.. _`lmc_only_for_last`:
+
+lmc_only_for_last
+-----------------
```

Comment:
I'm not a fan of the name `lmc_only_for_last`. If you expand it, it reads "Last message correction only for last" which is weird and it also requires people to know what "lmc" means.

How about `allow_message_corrections` which takes 3 values: `'all'`, `'last'` or anything falsy?